utils/obsproc/util.h

```diff
@@ -218,6 +218,24 @@ namespace gdasapp {
           oops::Log::test() << checksum(latitude_, "latitude") << std::endl;
           oops::Log::test() << checksum(datetime_, "datetime") << std::endl;
         }
+
+       // Swap the date
+       void reDate(int64_t minDAwindow, int64_t maxDAwindow, int windowcyc) {
+         for (int i = 0; i < location_; i++) {
+           if (datetime_(i) < minDAwindow) {
+             int timelatency = minDAwindow - datetime_(i);
+             datetime_(i) = minDAwindow + 1;  // for safety 1 sec
+             obsError_(i) += 0.1 / (3600 * windowcyc) * timelatency;
+           } else if (maxDAwindow < datetime_(i)) {
```

Comment:
no need for an `else if` ... just add an other `if` statement.

Reply:
Done.